site/ultraviolet/assets/semantic-ui/js/ultraviolet/ultraviolet.deposit.js

```diff
@@ -0,0 +1,14 @@
+import { waitForElement } from "./waitForElement";
+
+document.addEventListener('DOMContentLoaded', async () => {
+
+  const fileUploadWarning = await waitForElement("div.ui.visible.warning.message")
```

Comment:
Here, you are still using the selector ```div.ui.visible.warning.message``` instead of ```.file-upload-note .ui.visible.warning.message```. Is there a reason not to use a more specific and targeted selector?

I see that you are assigning ```1``` as the id of ```secondWarning``` node. The element you are cloning as template does not have id (```<div class="ui visible warning message">``` making the assignment of ```fileUploadWarning.id + "1"``` to be  ```1```.  Ids must not start with numbers (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)

Also, since you are cloning and replacing text, why not just use the markup from https://semantic-ui.com/collections/message.html?

You can try:

```javascript
  const fileUploadWarning = await waitForElement(".file-upload-note div");
  if (fileUploadWarning) {
    const message = `Contact the NYU Libraries UltraViolet service team at <a href="mailto:uv@nyu.edu">uv@nyu.edu</a> if you have more than 100 files or 50GB.`
    const div = document.createElement("div");
    div.id = "invenio-84-file-limits-msg";
    div.classList = "ui visible warning message";
    div.innerHTML = `<p><i aria-hidden="true" class="warning sign icon"></i>${message}</p>`
    fileUploadWarning.append(div);
  }
```

Reply:
Thanks for the tip.
- RE:selector - Added changes to use a more specific selector. 
- RE:id - Was trying to qualm the react lords with duplicate elements and id's, but your solution is cleaner, so I used that. 
I also went ahead and separated per ticket solution so that in the event we need to backtrack these changes we can reference the ticket and use that to solve technical debt. 
Should be good to go.